src/components/accordion/accordion-header.tsx

```diff
@@ -23,15 +24,6 @@ const AccordionTitleHeadline = styled.span`
   color: inherit;
 `;
 
-const resetButton = css`
-  padding: 0;
-  margin: 0;
-  appearance: none;
-  background: 0;
-  border: 0;
-  box-shadow: none;
-`;
-
```

Comment:
I chose to make this reset available for sharing to use in other button-like components.